hybrid/utils.py

```diff
@@ -64,10 +64,10 @@ def bqm_reduced_to(bqm, variables, sample, keep_offset=True):
         This example reduces a 3-variable BQM to two variables.
 
         >>> import dimod           # Create a binary quadratic model
-        >>> bqm = dimod.BinaryQuadraticModel({}, {('a', 'b'): -1, ('b', 'c'): -1, ('c', 'a'): -1}, 0, 'BINARY')
+        >>> bqm = dimod.BinaryQuadraticModel({}, {('a', 'b'): -1.0, ('b', 'c'): -1.0, ('c', 'a'): -1.0}, 0.0, 'BINARY')
```

Comment:
Why is this necessary for doctests to work?

Reply:
doctest was failing because when the biases are set as int that's preserved so it was barfing on the float results

Reply:
I don't understand what do you mean by that.

In this case for integer couplings, couplings in the reduced bqm are also integers.

In other words, there's no need to upcast the input, you can just use integer output:
```
BinaryQuadraticModel({'b': 0, 'a': 0}, {('b', 'a'): -1}, 0, Vartype.BINARY)
```

Reply:
I actually answered my initial question with that last comment. To match (mixed) float output that was already present in the docstring, you upcasted everything to float.

That's fine. Although unnecessary. :smile: